.github/workflows/validate-json.yml

```
@@ -0,0 +1,16 @@
+name: 'Validate JSON'
+
+on:
+  push:
+    paths:
+      - '**/*.json'
```

```suggestion
  push:
    paths:
      - '**/*.json'
  pull_request:
    paths:
      - '**/*.json'
```

This will run it on PRs from forks, too.

---

Oh cool